README.md

```diff
@@ -59,10 +59,18 @@ CmdStan path set to: {a path on your file system}
 
 If you run into trouble, consult the official [`cmdstanr`](https://mc-stan.org/cmdstanr/index.html) website for further installation guides and help.
 
-## Download this repository and install the project package (`wwinference`)
+## Install `wwinference`
+
 Once `cmdstanr` and `CmdStan` are installed, the next step is to download this repository and install the package, `wwinference`. The package provides tools for specifying and running the model, and installs other needed dependencies.
 
-Once you have downloaded this repository, navigate to it within an R session and run the following:
+### User install
+
+The package can be installed directly from github by running the following within an R session:
+```R
+remotes::install_github("CDCgov/ww-inference-model")
+```
+### Developer install
+If you are intersted in contributing to the `wwinference` clone the repository, navigate to it within an R session and run the following:
 
 ```R
 install.packages('remotes')
```

Comment:
```suggestion
```

The earlier instructions also require `remotes` to be installed.

Reply:
Added it above to user install